labelImg.py

```diff
@@ -1102,7 +1128,7 @@ def load_file(self, file_path=None):
                 self.label_list.setCurrentItem(self.label_list.item(self.label_list.count() - 1))
                 self.label_list.item(self.label_list.count() - 1).setSelected(True)
 
-            self.canvas.setFocus(True)
```

Comment:
**Finding:** The setFocus function expects an enum, not a bool.
I looked up the corresponding value to True == 1 == Qt.TabFocusReason, so the behavior should be unchanged.

However, you might want to check out whether this does what you want it to do.

**To do:** 

- [ ] check if the enum value is what's intended here.